extras/otel/otel-collector-config.yaml

```diff
@@ -32,12 +32,18 @@ receivers:
           static_configs:
             - targets: ["0.0.0.0:8888"]
   docker_stats:
-    endpoint: unix:///var/run/docker.sock
-    timeout: 20s
-    api_version: 1.41
+    collection_interval: 1m
+    container_labels_to_metric_labels:
+      shuttle.idle_minutes: shuttle.idle_minutes
+      shuttle.project: shuttle.project_name
```

Comment:
Should this be `shuttle.project.name`? And perhaps the same convention for the project id and idle minutes, but we haven't implemented that elsewhere in our code.

Reply:
Agreed, I'll make this change